media/js/mozorg/home/home.js

```diff
@@ -2,147 +2,171 @@
                     // becomes percivable as the user scrolls down
-                    $stickyCTA.removeAttr('aria-hidden');
-                } else {
+                     $stickyCTA.removeAttr('aria-hidden');
+                 } else {
                     // hidden again as they scroll up
-                    $stickyCTA.attr('aria-hidden', 'true');
-                }
-            }
-        });
-
-        // add button
-        var $dismissButton = $('<button>').addClass('sticky-dismiss').text('Dismiss download prompt.');
-        var $stickyWrapper = $stickyCTA.find('.primary-wrapper');
-        $dismissButton.appendTo($stickyWrapper);
-        // listen for click
-        $dismissButton.one('click', function(){
+                     $stickyCTA.attr('aria-hidden', 'true');
+                 }
+             }
+         });
+
+         // add button
+         var $dismissButton = $('<button>').addClass('sticky-dismiss').text('Dismiss download prompt.');
+         var $stickyWrapper = $stickyCTA.find('.primary-wrapper');
+         $dismissButton.appendTo($stickyWrapper);
+         // listen for click
+         $dismissButton.one('click', function(){
             // dismiss
-            dismissStickyCTA(primaryTop);
-        });
-    }
-
-    // handle dismiss
-    function dismissStickyCTA(stickyWayPoint) {
-        // destroy waypoint
-        stickyWayPoint.destroy();
-        // remove element
-        $stickyCTA.remove();
-        // set cookie, if cookies are supported
-        if (hasCookies) {
-            var d = new Date();
-            d.setTime(d.getTime() + (30 * 24 * 60 * 60 * 1000)); // 30 days
-            Mozilla.Cookies.setItem('sticky-home-cta-dismissed', 'true', d, '/');
-        }
-    }
-
-    // Check if previously dismissed
-    if (hasCookies) {
-        if (!Mozilla.Cookies.getItem('sticky-home-cta-dismissed')) {
-            // init the button
-            initStickyCTA();
-        } else {
-            $stickyCTA.remove();
-        }
-    } else {
-        $stickyCTA.remove();
-    }
-
-})(window.Waypoint);
+             dismissStickyCTA(primaryTop);
+         });
+     }
+
+     // handle dismiss
+     function dismissStickyCTA(stickyWayPoint) {
+         // destroy waypoint
+         stickyWayPoint.destroy();
+         // remove element
+         $stickyCTA.remove();
+         // set cookie, if cookies are supported
+         if (hasCookies) {
+             var d = new Date();
+             d.setTime(d.getTime() + (30 * 24 * 60 * 60 * 1000)); // 30 days
+             Mozilla.Cookies.setItem('sticky-home-cta-dismissed', 'true', d, '/');
+         }
+     }
+
+     // Check if previously dismissed
+     if (hasCookies) {
+         if (!Mozilla.Cookies.getItem('sticky-home-cta-dismissed')) {
+             // init the button
+             initStickyCTA();
+         } else {
+             $stickyCTA.remove();
+         }
+     } else {
+         $stickyCTA.remove();
+     }
+
+ })(window.Waypoint);
```

Comment:
It looks like the sticky CTA on this page still uses jQuery :(

In the console I see the following error on page load:

```
Error: No element option passed to Waypoint constructor
```

I think this is because you're missing `window.jQuery` here as the first argument passed to the IIFE.